dashboards/google-gce-disk/gce-disk.json

```
@@ -0,0 +1,56 @@
+{
+  "displayName": "Disks",
+  "mosaicLayout": {
+    "columns": 48,
+    "tiles": [
+      {
+        "width": 48,
+        "height": 35,
+        "widget": {
+          "title": "disk bytes used",
```

```suggestion
          "title": "Disk bytes used",
```

Should the title be sentence case?